templates/includes/sidebar.html

```diff
@@ -1,4 +1,11 @@
 <ul class="sidebar-nav">
+   {% if user.is_authenticated %}
+    <li><a href="{% url "taxi:driver-detail" pk=user.id %}">User: {{ user.username }}</a></li>
```

Comment:
Ensure that the `driver-detail` URL pattern is correctly defined and that it accepts a `pk` parameter. This link will direct authenticated users to their driver detail page.